src/styles/variables/_colors.scss

```diff
@@ -38,8 +38,11 @@ $gray-dark: mix(black, $gray, $gray-light-ratio);
 $gray-darkest: mix(black, $gray, $gray-lightest-ratio);
 $black: black;
 $white: white;
-
+$alice-blue: #E8F4FD;
```

Comment:
i'd go with $status-blue or something like that.

Reply:
i actually have mixed feelins about this, but i think htat naming convetion lets us know more about what these colors are, and to not really use them outside of their purpose.

Reply:
Updated. Ready for another look @martypowell

Reply:
youll want to hit hit the other status colors too.

Reply:
Did a big cleanup. Pushing up. @martypowell